RELEASE.md

```diff
@@ -7,6 +7,8 @@
 1. Build the package locally:
 
    ```bash
+   # clean build artefacts to avoid issues
+   yarn run clean
```

Comment:
Just FYI, the clean command doesn't currently clear the `dist/` directory, and may be missing other directories that it should clear

Reply:
but the `yarn run webpack:prod` or whatever that is cleans `dist`.
We can do a `clean:slate` instead if one wants to be safe

Reply:
because people not always follow steps in a linear way and mess up when running through things  🤷🏽‍♀️

Reply:
That's true. But I would hope that someone would be a bit more careful when doing a release.

Reply:
I can remove that if it is too much cruft, as long as it makes sense to people it is fine. 
And I also hope people are careful, including me, but mistakes can happen, it's just the way it is.